docarray/predefined_document/audio.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional, TypeVar
+
+from docarray.document import BaseDocument
+from docarray.typing import AudioUrl, Embedding
+from docarray.typing.tensor.audio.audio_tensor import AudioTensor
+
+T = TypeVar('T', bound='Audio')
+
+
+class Audio(BaseDocument):
+    """
+    Document for handling audios.
+
+    The Audio Document can contain an AudioUrl (`Audio.url`), an AudioTensor
+    (`Audio.tensor`), and an Embedding (`Audio.embedding`).
+
+    EXAMPLE USAGE:
+
+    You can use this Document directly:
+
+    .. code-block:: python
+
+        from docarray import Audio
+
+        # use it directly
+        audio = Audio(url='https://www.kozco.com/tech/piano2.wav')
```

Comment:
Maybe we should rely on a file from github rather than a file from the internet ?
how about using the files here ? https://github.com/docarray/docarray/tree/feat-add-audio-v2/tests/toydata